### ISSUE TITLE
Upgrade to mermaid version 0.4.0 

### ISSUE THREAD

**Issue by jmatsushita** [cldwalker/Mermaid#1]
Hi there,

Very interesting plugin! I could make some of the online examples to work (for instance using the `graph TR` statement, so I tried to manually update the mermaid.full.min.js from the latest 0.4.0 release.... but it's not that simple it looks like!

Hope you get some time to upgrade it!

Best,

Jun

**Comment by cldwalker** [comment]
Thanks for reporting your issue! This is [one of my 14 active issues](https://gh-active-issues.herokuapp.com/#cldwalker/Mermaid#1). Use that link to check how soon your issue will be answered. Don't forget to check your issue against this project's CONTRIBUTING.md. Cheers.

**Comment by cldwalker** [comment]
Yeah. Unfortunately upgrading isn't straightforward because their api is in flux. Since I don't use this plugin much, happy to take a pull from you or anyone else who digs more into this
